src/tools/rbd/action/Group.cc

```diff
@@ -82,6 +82,31 @@ std::string get_group_snap_state_name(rbd_group_snap_state_t state)
   }
 }
 
+std::string get_group_snap_state_name2(cls::rbd::GroupSnapshotState state)
```

Comment:
Why is this name2?

Reply:
This is not required, the previous command is fine.
The CLI uses the librbd calls, it should not require to use the cls::rbd calls directly.